inc/providers/class-core-sitemaps-posts.php

```diff
@@ -68,7 +68,19 @@ public function get_url_list( $page_num, $post_type = '' ) {
 			return array();
 		}
 
-		$query = new WP_Query(
+		/**
+		 * Filters the post type URL list query arguments.
+		 *
+		 * Allows modification of the URL list query arguments before querying.
+		 *
+		 * @see WP_Query for a full list of arguments
+		 *
+		 * @since 5.5.0
+		 *
+		 * @param array $args An array of WP_Query arguments.
+		 */
+		$args = apply_filters(
+			'core_sitemaps_post_url_list_query_args',
```

Comment:
Depending on whether #182 gets in first, we'd need to rename this appropriately.

Reply:
I removed it first, but then the PHPCS checks failed, so I re-added the `core_` prefix. Perhaps we should remove the check from the phpcs config.

Reply:
That's what #182 is dealing with.

Reply:
should I remove it?

Reply:
No, you can leave it as-is.

What I wanted to say is that we need to rename it if and only if #182 gets merged before this PR.